packages/api-elements/lib/define-value-of.js

```diff
@@ -113,8 +121,10 @@ function mapValue(e, options, f, elements) {
   }
 
   const opts = updateTypeAttributes(e, options);
-  if (e.content && (!isPlural(e) || !e.isEmpty)) {
+
+  if (e.content && !(isPlural(e) && (e.isEmpty || e.content.every(isPrimitiveAndHasNoValue)))) {
```

Comment:
nit: I get this is does unnecessary e.content.every() call on empty content but I am trying to make all the conditions more readable

```suggestion
  if (e.content && !(isArrayElement(e) && e.content.every(isNoValuePrimitive))) {
```

Reply:
Fair